### ISSUE TITLE
UV and vertex colour splits are not accounted for.

### ISSUE THREAD

**Issue by FrankieHobbins** [Vinc3r/Polycount#26]
When you load a model in a game engine, the vert count gets increased when there are uv splits or faces with different vertex colours to neighbouring faces.

This plugin doesn't seem to account for these.

**Comment by Vinc3r** [comment]
yep it doesn't detect these data yet. no idea when i will have the skills need for this :)
